src/test/resources/projects/standard-project/pom.xml

```diff
@@ -14,30 +14,79 @@
   <packaging>war</packaging>
   <name>App Engine Standard Project</name>
 
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>com.google.cloud.tools</groupId>
-        <artifactId>appengine-maven-plugin</artifactId>
-        <version>${appengine-maven-plugin.version}</version>
-        <configuration>
-          <version>1</version>
-          <project>travis-app-maven-plugin</project>
-          <version>GCLOUD_VERSION</version>
-          <jvmFlags>
-            <!-- required for kokoro gcp_ubuntu b/37217450 -->
-            <jvmFlag>-Djava.security.egd=file:/dev/urandom</jvmFlag>
-          </jvmFlags>
-          <environment>
-            <TEST_VAR>testVariableValue</TEST_VAR>
-          </environment>
-          <project>travis-app-maven-plugin</project>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
 
   <profiles>
+    <profile>
+      <id>java8_and_below</id>
```

Comment:
Would this count as a breaking change? Since we won't be able to run just `mvn package appengine:<goal>` (i.e without the profile) anymore.

Reply:
This is actually only a problem when running dev server. It's not related to the maven plugin at all.
This project has a `RunAsyncMojoIntegrationTest.testRunAsync()` test that starts the devserver and verifies that it starts successfully.

I have a standalone example here -- https://github.com/elefeint/examples/tree/master/appengine17-helloworld

Reply:
You are right that we should add this to the user guide, since `mvn appengine:run` will have this issue on Java 16 and up.